ui/src/App.vue

```diff
@@ -158,6 +158,6 @@ body, #app {
   background: rgba(0, 0, 0, .5)
 }
 .danger {
-  color: red;
+  color: red !important;
```

Comment:
Noticed this while I was here - style for delete button was being overridden by mdl